.github/workflows/used-by.yml

```
@@ -21,6 +21,7 @@ jobs:
         with:
           add-paths: "README.md" # the file path to commit
           commit-message: "chore: update used-by badge by github-actions[bot]"
+          committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
```

this is using the default value.

If you don't want attribution, then I think you can force the author to be the bot:
```suggestion
          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
```